spec/machine/MachineExports.thy

```diff
@@ -67,32 +67,38 @@ requalify_consts
   timerPrecision
   max_time
   getCurrentTime_buffer
+  \<mu>s_in_ms
 
 requalify_facts
-  us_to_ticks_mono
-  us_to_ticks_zero
-  us_to_ticks_nonzero
-  kernelWCET_ticks_no_overflow
   MAX_PERIOD_US_def
   MAX_PERIOD_def
-  us_to_ticks_mult
-  getCurrentTime_buffer_no_overflow
   kernelWCET_ticks_def
   replicate_no_overflow
-  getCurrentTime_buffer_nonzero
   getCurrentTime_buffer_nonzero'
   getCurrentTime_buffer_no_overflow'
   getCurrentTime_buffer_no_overflow'_stronger
   getCurrentTime_buffer_minus
   getCurrentTime_buffer_minus'
   MAX_PERIOD_mult
-  MAX_PERIOD_mult'
+  ticks_per_timer_unit_non_zero
+  MIN_BUDGET_bound
+  getCurrentTime_buffer_bound
+  kernelWCET_pos'
+  MIN_BUDGET_pos'
+  domain_time_pos
+  getCurrentTime_buffer_pos
+  us_to_ticks_def
+  getCurrentTime_buffer_no_overflow
+  us_to_ticks_mono
+  MIN_BUDGET_helper
+  \<mu>s_in_ms_def
```

Comment:
This one, too?

Reply:
This is needed to show that \<mu>s_in_ms is equivalent to `usInMs`. This is in the same boat as `MAX_RELEASE_TIME` in needing to be consolidated somewhere appropriate. Shall I leave it in here for now and make a JIRA issue for this constant business?

Reply:
I see, that’s fair enough. Yes, making a JIRA issue would  be great.

Reply:
https://sel4.atlassian.net/browse/VER-1411